Filters/exclusions.txt

```diff
@@ -1,5 +1,7 @@
 ! https://github.com/AdguardTeam/AdguardFilters/issues/39674 
 ||stats.lt^
+! blocked by DNS
```

Comment:
what reason to unblock it?

Reply:
According the first user response `get server not found from apps used mybluemix.net`, but I asked our support team learn about some details and didn't receive it from user for now. Please don't close this pr, maybe we get similar issues in future with more details.